sci_bert/data_readers/conll_data_reader.py

```diff
@@ -0,0 +1,173 @@
+from typing import Dict, List, Sequence, Iterable
```

Comment:
do we need to copy the reader here or can we refer to the one from allennlp in the config file ?

Reply:
The reason was to make adjustments for handling BERT which might not be possible through only config file.